app/views/item_seasons/index.html.erb

```diff
@@ -1,9 +1,10 @@
-<h1 class="page__title"> <%= I18n.t("item_season.index.title") %> </h1>
+<div class="content background-color-blue">
+  <h1 class="page__title banner-small"> <%= I18n.t("item_season.index.title") %> </h1>
 
-<table class="produce-table">
-  <% ItemSeason::MONTHS.each do |month_index| %>
-    <tr>
-      <td class="produce-table__month"><%= link_to (formatted_month_name(month_index)).capitalize, item_season_path(month_index), class: "link_action month_link" %></td>
-    </tr>
-  <%  end %>
-</table>
+  <div class="button-holder p-top-l p-bottom-l p-sides-s">
+    <% ItemSeason::MONTHS.each do |month_index| %>
+      <%= link_to (formatted_month_short(month_index)).capitalize, item_season_path(month_index),
+          class: "link_action month_link button button--yellow script-lg m-top-s m-bottom-s #{randomized_rotation}" %>
```

Comment:
**nitpick:** this is not relevant at this scale, so feel free to ignore this comment. I'm leaving it here just so you are generally aware of this type of issue.

Here you are calculating something random 12 times. Not sure how Ruby handles this and I'm sure the performance hit is negligible even in cases that you calculate this 100mi times. That said, it's always worthwhile to consider these questions (like, what's the impact of calculating a value on every loop iteration).

A more performance approach would be doing some deterministic math using the month index. But, then I don't think it's worth your time to worry about an alternative here.

Reply:
At first, when designing the layout, I thought I would make specific rotations for specific months. But while working I thought the random rotations might be interesting. I think always changing is interesting, but it can look uglier, and there is the performance side of things.

So i made a version with the 'fixed' rotations. WDYT?

Reply:
Sure, makes sense. The original way was fine as well, I just took the opportunity to expose aspects you should consider in more complex cases (as of indicating a certain mindset to have).

Reply:
I think it is a great exercise in weighting pros and cons of different solutions, and deciding what makes more sense. To think beyond our first solution. I think I'm going with our second solution, I think the greater control of what gets rendered is nice.